kmip/core/attributes.py

```diff
@@ -63,12 +63,35 @@ class NameValue(TextString):
         def __init__(self, value=None):
             super(Name.NameValue, self).__init__(value, Tags.NAME_VALUE)
 
+        def __eq__(self, other):
+            if isinstance(other, Name.NameValue):
+                if self.value == other.value:
+                    return True
+            return False
```

Comment:
The other instances where __eq__ is implemented the final return statement returns NotImplemented for some reason. You would need to ask Peter why that is.